logrotate.c

```diff
@@ -2146,7 +2146,8 @@ static int rotateLogSet(struct logInfo *log, int force)
 			"since no logs were rotated\n");
 	    } else {
 		message(MESS_DEBUG, "running postrotate script\n");
-		if (runScript(log, log->flags & LOG_FLAG_SHAREDSCRIPTS ? log->pattern : log->files[j], log->post)) {
+		// It only makes sense to pass in a final rotated filename if scripts are not shared
```

Comment:
Please use `/* ... */` comments instead of `//` comments, which are used nowhere else in logrotate code.